### PR TITLE
Reinstate bad squidnames

### PR DESCRIPTION
**Pull request by Effectism**
## About The Pull Request

Why were these ever removed?

## Changelog
:cl:
add: Added 2 default squid names
/:cl:

### CODE REVIEW

**Comment by Mick1299** [comment]
you changes the lizard female name, not the squid female name

File: code/_globalvars/lists/names.dm [hunk 1]
```diff
@@ -12,9 +12,11 @@ GLOBAL_LIST_INIT(first_names_male, world.file2list("strings/names/first_male.txt
 GLOBAL_LIST_INIT(first_names_female, world.file2list("strings/names/first_female.txt"))
 GLOBAL_LIST_INIT(last_names, world.file2list("strings/names/last.txt"))
 GLOBAL_LIST_INIT(lizard_names_male, world.file2list("strings/names/lizard_male.txt"))
-GLOBAL_LIST_INIT(lizard_names_female, world.file2list("strings/names/lizard_female.txt"))
-GLOBAL_LIST_INIT(squid_names_male, world.file2list("strings/names/squid_male.txt"))
+GLOBAL_LIST_INIT(lizard_names_female, world.file2list("austation/strings/names/lizard_female.txt"))
```
Comment (Mick1299) [inline code/_globalvars/lists/names.dm:15]: This one should stay as strings/name/lizard_female.txt

File: code/_globalvars/lists/names.dm [hunk 2]
```diff
@@ -12,9 +12,11 @@ GLOBAL_LIST_INIT(first_names_male, world.file2list("strings/names/first_male.txt
...
+//Austation begin -- Enriched selection of squid names
+GLOBAL_LIST_INIT(squid_names_male, world.file2list("austation/strings/names/squid_male.txt"))
 GLOBAL_LIST_INIT(squid_names_female, world.file2list("strings/names/squid_female.txt"))
```
Comment (Mick1299) [inline code/_globalvars/lists/names.dm:18]: This is the one that should change to austation/strings/names/squid_female.txt

File: code/_globalvars/lists/names.dm (hunk 1 above)
Comment (Mick1299) [inline code/_globalvars/lists/names.dm:15]: This needs to be changed back to "strings/names/lizard_female.txt"

File: code/_globalvars/lists/names.dm (hunk 2 above)
Comment (Mick1299) [inline code/_globalvars/lists/names.dm:18]: and this one needs to change to "austation/strings/names/squid_female.txt"

Comment (Effectism) [reply code/_globalvars/lists/names.dm]: fixed, thanks

File: code/_globalvars/lists/names.dm (hunk 1 above)
Comment (Effectism) [reply code/_globalvars/lists/names.dm]: fixed, thanks

**Comment by jupyterkat** [comment]
image
continue being based my man

**Pull request merged**